riotfile.py

```diff
@@ -2533,6 +2533,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="anthropic",
+            command="pytest {cmdargs} tests/contrib/anthropic",
+            pys=select_pys(min_version="3.7", max_version="3.11"),
```

Comment:
Just curious, is there a reason we don't support anthropic with python 3.12?

Reply:
ill change this